src/engine/renderer/tr_image.cpp

```diff
@@ -3049,6 +3049,13 @@ int numTextures = 0;
 
 qhandle_t RE_GenerateTexture( const byte *pic, int width, int height )
 {
+	size_t size = width * height;
+
+	ASSERT( !size );
```

Comment:
I thought ASSERT fails if the condition is false?

Reply:
Fixed.